android/src/main/java/com/mapbox/mapboxgl/MapboxMapController.java

```diff
@@ -738,4 +756,60 @@ private int checkSelfPermission(String permission) {
       permission, android.os.Process.myPid(), android.os.Process.myUid());
   }
 
+  /**
+   * Tries to find highest scale image for display type
+   * @param symbol
+   * @param density
+   * @return
+   */
+  private Bitmap getImage(Symbol symbol, float density) {
```

Comment:
Can we optimize the naming so the intent of the method becomes clear for the reader? eg. getScaledImage, getDensityOptimisedImage, loadDensityDependentImage, ..

Reply:
Renamed to getScaledImage.